### PR TITLE
introduce config --services, --volumes, --hash for backward compatibility

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
Implemented --services, --volumes, --hash pseudo-commands on `compose config`

this will prevent compatibility issues with docker-compose

**Related issue**
close https://github.com/docker/compose-cli/issues/1442

### CODE REVIEW

File: utils/hash.go [hunk 1]
```diff
@@ -14,27 +14,21 @@
...
-}
-
-func contains(slice []string, item string) bool {
-	for _, v := range slice {
-		if v == item {
-			return true
-		}
-	}
-	return false
+	return digest.SHA256.FromBytes(bytes).Hex(), nil
```
Comment (gtardif) [inline utils/hash.go:33]: why this change to Hex ? 
And then I just noticed `// Hex is deprecated. Please use Digest.Encoded.`

Comment (ndeloof) [reply utils/hash.go]: updated to use Encoded()
String() do include the algorithm (`sha256:...`), so aligned with docker-compose
